network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/wide_table_block.html

```diff
@@ -2,41 +2,31 @@
 {% load wagtailcore_tags table_block_tags %}
 
 {% block block_content %}
-    <div class="article-table-block table-responsive">
+    <div class="article-table-block table-responsive{% if not self.wide %} tw-mx-auto tw-max-w-[80%]{% endif %}">
```

Comment:
Hi @robdivincenzo! 

I noticed that on medium breakpoints and below, the `max-w-80%` class is making the table narrower than the rest of the content. 

<img width="1401" alt="Screenshot 2024-09-10 at 12 13 11 PM" src="https://github.com/user-attachments/assets/5c9f83ec-f39c-42e5-99a0-69e6ceacd8c6">



Can we double-check with design if that’s what is expected?

If the table should match the width of the surrounding content at those breakpoints, updating the class like this:

```suggestion
    <div class="article-table-block table-responsive{% if not self.wide %} tw-mx-auto medium:tw-max-w-[80%]{% endif %}">
```

should do the trick 👍 thanks!

Reply:
Great catch @danielfmiranda ! I've updated the template with the new targeted viewport. Needed to narrow on`large` to cover it.